db/migrate/20230119074348_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :users, null: false, foreign_key: true
```

Comment:
- [ ]  according to project requirement kindly reference user with author i.e `t.references :author, foreign_key: { to_table: 'users' } `